template.go

```diff
@@ -346,6 +351,7 @@ func newTemplate(name string) *template.Template {
 		"keys":          keys,
 		"last":          arrayLast,
 		"replace":       strings.Replace,
+		"parseBool":     parseBool,
```

Comment:
You may just want to do `"parseBool": strconv.ParseBool` here since the function signature and return type isn't changing.